db/single/reference/jpeg/generating.py

```diff
@@ -20,6 +20,23 @@
 img = Image.open('../../jpeg/134/site-1534685_1280.jpg')
 np.save('site-1534685_1280', np.asarray(img))
 
-img = img.crop((20, 5, 1000, 800))
-np.save('site-1534685_1280_roi', np.asarray(img))
+cropped = img.crop((20, 5, 1000, 800))
+np.save('site-1534685_1280_roi', np.asarray(cropped))
+
+def rgb_to_ycbcr(rgb):
+    ycbcr_weights = np.asarray([[0.257, 0.504, 0.098],
+                                [-0.148, -0.291, 0.439],
+                                [0.439, -0.368, -0.071]]).T
+    ycbcr = np.matmul(rgb, ycbcr_weights) + np.asarray([0.0625, 0.5, 0.5])
+    return ycbcr
```

Comment:
Here is a funny thing: I just took those from @szkarpinski's [code](https://github.com/NVIDIA/DALI_extra/blob/main/db/imgcodec/colorspaces/make.py), and those match numers in DALI [kernel](https://github.com/NVIDIA/DALI/blob/main/dali/kernels/imgproc/color_manipulation/color_space_conversion_impl.h#L67). I can't see these numbers in the [paper](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.601-7-201103-I!!PDF-E.pdf) itself.

Reply:
Ok, I asked this because I couldn't find these numbers anywhere in the standard or something and it is not clear to me where they come from. I think it would be good to find it out.
Maybe @jantonguirao can help here?

Reply:
They come from https://en.wikipedia.org/wiki/YCbCr#ITU-R_BT.601_conversion

Particularly from this expression:
![image (10)](https://user-images.githubusercontent.com/3891217/185916549-1f08d514-aa35-4f0b-88e1-98aa8f6debc5.png)

@staniewzki you can add the link to wikipedia as a comment

Reply:
@awolant Note: the biases are normalized to 1 in the code, since the inputs are float in the range 0..1